### PR TITLE
Feature | Pages Navigation

### PR DESCRIPTION
**Pull request by errea**
## Desription
In this project, we worked on the Awesome books application and had to convert it into a fully functional website. We refactored the project to add three new sections on top of the two sections you have already completed: 

## Project requirements [A few checks passed]
- [x] A Navigation bar.
- [x] Books list.
- [x] Add book form.
- [x] Contact info.
- [x] Make sure that when you click on any of the links, the main area changes to display only one of the three content sections.
- [x] No library or framework, use only plain JavaScript and CSS
- [x] Don't use any library or framework, use only plain JavaScript.
- [x] Make sure that when you click on any of the links, the main area changes to display only one of the three content 
- [x] Display the current date below the navigation bar. For this requirement, you should use a library - Luxon
 
> ## Checklist:
 - [x] Our code follows the style guidelines of this project
 - [x] We performed some code checks on the LocalStorage
 - [x] We have commented on the code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is having one of the linter checks failing. Kindly ensure that all linter checks are running with no errors before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: awesome-book.js [hunk 1]
```diff
@@ -55,6 +69,36 @@ class Library {
...
+
+function displayContact() {
+  const listSection = document.querySelector('#book-display');
+  const formSection = document.querySelector('#book-form-section');
+  const contactSection = document.querySelector('#contact-section');
+
+  listSection.style.display = 'none';
+  formSection.style.display = 'none';
+  contactSection.style.display = 'flex';
+}
```
Comment (ZawadiSumba66) [inline awesome-book.js]: - [x] The rendering of different content when one of the links is clicked currently contains repeated code. I would like you to consider if you had 5 tabs or more... wouldn't it contain large amounts of code and probably make it difficult to debug? Thanks to looping...
Hint: You can loop through all the links and then render the content dynamically. Remember, using a DRY rule in your code makes it easy to read and debug.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi  @errea  && @alvp01 👋 👋 , 

Good job so far!

### HIGHLIGHTS 🚀 🚀 
- Great UI design ✔️ 
- Descriptive pull request and README ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: awesome-book.js (hunk 1 above)
Comment (alvp01) [reply awesome-book.js]: The iterative solution is confusing to me. I thought of a solution using an array with display properties corresponding to each section depending o which link was clicked

Comment (alvp01) [reply awesome-book.js]: Solved the repeated code problem with a single function and the use of an array. Hope this time it gets approved.

**Review by hemantso — APPROVED** [review]
Hi @errea and @alvp01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Everything is working fine. Try to add `live link` in your README file.


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alvp01** [comment]
> Hi @errea and @alvp01,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> ## Congratulations! tada
> 
> Alt Text
> ## Optional suggestions
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
>     * Everything is working fine. Try to add `live link` in your README file.
> 
> 
> Cheers,
> Hemant Soni
> Cheers and happy coding smiley
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks for your time and work! I added the live link to the project as you suggested. Happy weekend.

**Pull request merged**